bin/pycbc_make_skymap

```diff
@@ -448,6 +461,8 @@ if __name__ == '__main__':
                         action=MultiDetOptionAppendAction,
                         help='Lists of local frame files, e.g., '
                              'H1:/path/to/frame/file L1:/path/to/frame/file')
+    parser.add_argument('--injection-file', type=str,
+                        help='The injection file as an .xml.')
```

Comment:
Does anything actually prevent the use of HDF injection files? If not, please update the help text to something less specific, like "Optional path to an injection file".

Reply:
I'm using `pycbc_create_injections` which, looking back to the example, does create a `injections.hdf` file, not an xml which I think I'm forcing it to do. Might explain some of the strange things it was doing when I was using it